l10n_it_fatturapa_out_rc/models/account_move.py

```diff
@@ -20,6 +20,7 @@ def generate_self_invoice(self):
                 doc_id = self.fatturapa_attachment_in_id.name
             else:
                 doc_id = self.ref if self.ref else self.name
+            self.rc_self_invoice_id.related_documents = [(5, False, False)]
```

Comment:
```suggestion
            self.rc_self_invoice_id.related_documents = [Command.clear()]
```
Non è bloccante, solo più leggibile